hal/armv7m/pmap.c

```diff
@@ -194,7 +195,9 @@ void _pmap_init(pmap_t *pmap, void **vstart, void **vend)
 	*(pmap_common.mpu + mpu_ctrl) |= (1 << 2);
 	hal_cpuDataMemoryBarrier();
 
-	for (i = 0; i < cnt; ++i) {
+	pmap_common.regionCnt = syspage->hs.mpu.allocCnt;
```

Comment:
if this change
`pmap_common.regionCnt = syspage->hs.mpu.allocCnt;`
is because syspage data may be overwritten, then what with:
`t = syspage->hs.mpu.table[i].rbar;` (line 201)
and
`t = syspage->hs.mpu.table[i].rasr & ~1;` (line 210)
syspage->hs mpu array
?

Reply:
It's still in init, before first dynamic memory allocation

Reply:
Yep, I know, but later you'll need this array to rebuild regions/subregions during `mmap` allocation (or it is not planned?)

Reply:
Not planned for now, if some feature is added, this might be changed

Reply:
Ack.